app.rb

```diff
@@ -56,12 +65,15 @@ def list_all_people
 end
 
 def list_rentals_by_person_id
+  puts @rentals
   print 'ID of person: '
   id = @input.read.to_i
-
   puts 'Rentals:'
   @rentals.each do |rental|
     puts "Date: #{rental.date}, Book '#{rental.book.title}' by #{rental.book.author}" if rental.person.id == id
   end
+  #   @saved_rentals.each do |rental|
+  #     puts "Date: #{rental.date}, Book '#{rental.title}' by #{rental.author}" if rental.id == id
+  #   end
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.